app/models/invoice_item.rb

```diff
@@ -3,4 +3,8 @@ class InvoiceItem < ApplicationRecord
 
   belongs_to :item
   belongs_to :invoice
+
+  validates :quantity, presence: true
+  validates :unit_price, presence: true
+  validates :status, presence: true
```

Comment:
I think we want to have something similar to invoice, where we validate that it comes in as one of the 3 pre-approved values